extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +138,466 @@ For example
-
-A **feature** is a specific instantiation of class containing **property values**. Property values are stored in either a **property table** or a **property texture** depending on the use case. Both formats are designed for storing property values for a large number of features.
-
-By default, properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. Built-in semantics include `ID` and `NAME`, as defined below.
-
-- `ID`: Unique identifier for the feature.
-- `NAME`: Name of the feature; not required to be unique.
-
-Model authors may define their own application- or domain-specific semantics separately. For application-specific semantics, authoring implementations are encouraged to use a `_*` prefix. For semantics common to a particular domain or vendor, creation of new uppercase, alphanumeric prefixes is encouraged.
-
-This extension implements the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata), which describes the metadata format in full detail.
-
-### Schemas
-
-A schema defines a set of classes and enums used in a model. Classes serve as templates for features - they provide a list of properties and the type information for those properties. Enums define the allowable values for enum properties. Schemas are defined in full detail in the [Schemas](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata#schemas) section of the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata).
-
-A schema may be embedded in the extension directly or referenced externally with the `schemaUri` property. Multiple glTF models may refer to the same external schema to avoid duplication.
-
-Schemas may be given a `name`, `description`, and `version`.
+### Overview
+
+Feature properties describe attributes or characteristics of a feature. Schemas describe the data types and semantic meanings of these properties. Schema definitions are generic, and do not describe the storage location or layout of the data.
+
+Conceptually, a feature is an instantiation of a generic template defined in the schema, with specific property values and storage information. Properties may be associated with features in one of two ways:
+
+- **Property Tables** store property values as numeric arrays in a parallel, column-based binary layout. Property tables are indexed by Feature IDs, used as the index of a given feature into each property array.
+- **Property Textures** store property values in channels of a texture, suitable for very high-frequency data mapped to less-detailed 3D surfaces. Property textures are indexed by texture coordinates, and do not have associated Feature IDs.
```

Comment:
Meta: should property textures be a separate extension?

Reply:
We talked about this, we think it's best to hold off for now until we submit a PR to Khronos to see how the broader community thinks these extensions should be organized.